.github/CODEOWNERS

```diff
@@ -0,0 +1,62 @@
+# Order is important; the last matching pattern takes the most precedence.
+
+.github            @alpenlabs/admins             # only admins should own the CODEOWNERS
+.gitignore         @alpenlabs/admins
+**/Cargo.toml      @alpenlabs/dependencies       # dependencies and metadata
+**/Cargo.lock      @alpenlabs/dependencies       # updates
+.config/           @alpenlabs/rust @alpenlabs/ci # config for rustdev and CI
+**/*.toml          @alpenlabs/rust               # config files
+
+# CI/Local ENV
+.github/workflows/     @alpenlabs/ci
+.github/actions/       @alpenlabs/ci
+.github/dependabot.yml @alpenlabs/ci
+Makefile               @alpenlabs/rust @alpenlabs/ci @alpenlabs/bin
+codecov.yml            @alpenlabs/ci
+contrib/               @alpenlabs/ci @alpenlabs/bin
+tests/                 @alpenlabs/rust @alpenlabs/ci
+
+# Docker/Containers
+**/Docker*         @alpenlabs/docker
+docker/            @alpenlabs/docker
+
+# Libs
+crates/bridge*     @alpenlabs/bridge
+crates/btc*        @alpenlabs/btc
+crates/common      @alpenlabs/rust
+crates/chaintsn    @alpenlabs/rust
+crates/consensus*  @alpenlabs/chain-core
+crates/db          @alpenlabs/database
+crates/eectl       @alpenlabs/evm
+crates/evm*        @alpenlabs/evm
```

Comment:
There's only one match for this: `crates/evmexec`. Why not specify it directly?

Reply:
I'm trying to future-proof it